casper/src/main/scala/coop/rchain/casper/blocks/proposer/Proposer.scala

```diff
@@ -197,7 +198,7 @@ object Proposer {
 
     val proposeEffect = (c: Casper[F], b: BlockMessage) =>
       // store block
-      BlockStore[F].put(b) >>
+      BlockStore[F].put(b.blockHash, b) >>
```

Comment:
Please create extension for `put(b: BlockMessage)` as it was defined on old BlockStore.

Reply:
[Done](https://github.com/rchain/rchain/pull/3655/commits/84c63c30b82730313a2a9deecd5a9b69dd0bb118).